README.md

```diff
@@ -241,6 +243,31 @@ VimeoClient.getInstance().fetchCurrentUser(new ModelCallback<User>(User.class) {
 });
 ```
 
+## FAQs
+### How do I play a video?
+Once you have a ```Video``` object you have two choices - embed or play natively. All consumers of this library will be able to access the embed data, however, in order to play videos natively (for example, in a VideoView or using ExoPlayer on Android) you must have access to the ```VideoFile``` links - this is restricted to users with Vimeo PRO accounts accessing their own videos.
```

Comment:
I would include a bit here specifying that a pro user can get a list of their videos through `fetchContent` to `User#getVideosConnection().uri` (on the result of fetchCurrentUser) or through hitting the `/me/videos` endpoint when the current PRO user is authenticated.
It might just clear up some ambiguity since that part's pretty crucial for pro users getting their vids